src/components/OrganizationsList/index.js

```diff
@@ -0,0 +1,27 @@
+import { connect } from 'react-redux';
+import {
+  getOrganizations,
+  getOrganizationsLoadingProps,
+} from '../../store/modules/organizations/selectors';
+import { loadOrganizations } from '../../store/modules/organizations/actionCreators';
+import Component from './Component';
+
+const mapStateToProps = () => {
+  return globalState => {
+    return {
+      ...getOrganizationsLoadingProps(globalState),
+      organization: getOrganizations(globalState),
+    };
+  };
+};
+
+export function mapDispatchToProps(dispatch) {
+  return {
+    loadOrganizations: () => dispatch(loadOrganizations()),
+  };
+}
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(Component);
```

Comment:
I would put `mstp` and `mdtp`  to the same page where we write our component, so that we clearly see what props come from where.